lib/is-map.js

```diff
@@ -0,0 +1,7 @@
+"use strict";
+
+function isMap(val) {
+    return (typeof Map !== "undefined" && val instanceof Map) || false;
```

Comment:
`(typeof Map !== "undefined" && val instanceof Map)` can only return `true` or `false`. The `|| false` can be omitted.